concepticondata/conceptlists.tsv

```diff
@@ -300,3 +300,4 @@ Hsiu-2015-315	Hsiu, Andrew C.	2015		315	areal, questionnaire	English	Hmong-Mien
 Maciejewski-2016-100	Maciejewski, Greg and Klepousniotou, Ekaterini	2016		100	ranked	English	English	 http://osf.io/7k3eh/	Maciejewski2016		The list comprises relative meaning frequencies for 100 homonyms (British eDom Norms) and other lexical norms. For feasibility, each line of the table was duplicated to map the different meanings to one of the concepts found in Concepticon. The meaning definition was taken from the Wordsmyth dictionary [(Parks et al. 1998)](:bib:Parks1998). The dominance scores were included from [Armstrong et al. (2012)](:bib:Armstrong2012). Word frequencies and number of orthographic neighbours  were based on the British National Corpus [(BNC 2007)](:bib:BNC2007). The numbers for word senses were taken from [Parks et al. (1998)](:bib:Parks1998) and WordNet [(Fellbaum 1998)](:bib:Fellbaum1998). Semantic diversity values were included from [(Hoffman et al. 2013)](:bib:Hoffman2013). Ratings on familiarity, concreteness, and imageability were taken from [Stadthagen-Gonzalez and Davis (2006)](:bib:Stadthagen2006) and [Coltheart (1981)](:bib:Coltheart1981).		
 Clark-2008-503	Clark, Eddie R.	2008		503	areal, questionnaire	English	Mienic languages	https://inter.payap.ac.th/wp-content/uploads/linguistics_students/Eddie_Clark_Thesis.pdf	Clark2008		The study includes two word lists, this concept list contains the lexical items in Kim Mun in Laos.		
 Clark-2008-441	Clark, Eddie R.	2008		441	areal, questionnaire	English	Mienic languages	https://inter.payap.ac.th/wp-content/uploads/linguistics_students/Eddie_Clark_Thesis.pdf	Clark2008		The study includes two word lists, this concept list contains the lexical items in Kim Mun in Vietnam.		
+Sawka-2019-201	Sawka, Kenneth S. and Mbewe, Christopher and Josephat, Daka and Schwertfeger, Jacob	2019		201	basic, areal, questionnaire	English	Toka-Leya variants and closely related Tonga languages	https://www.sil.org/resources/publications/entry/80878	Sawka2019		This word list was used for a lexicostatistical analysis of Toka-Leya in Zambia. 201 concepts were used to compare 11 languages and varieties, although the authors stated that there were 202 concepts. Almost all of the words were assigned to semantic categories.	44 – 52	
```

Comment:
Correct page number separator (xx-xx).

Reply:
Is this list based on something? What do the sources say?

Reply:
In the text, or at least in the sections that I read where one could expect to find a reference to the origin of the concepts, nothing was mentioned. The bibliography also did not have any references that would be as obvious as Swadesh or the like.